ui/admin/app/routes/scopes/scope/roles/role.js

```diff
@@ -40,4 +41,18 @@ export default class ScopesScopeRolesRoleRoute extends Route {
       this.router.replaceWith(transition.to.name, role.scopeID, role.id);
     }
   }
+
+  // =actions
+
+  /**
+   * Event to determine whether the loading template should be shown.
+   * Only show the loading template when transitioning into the specified route.
+   * @param transition
+   * @returns {boolean}
+   */
+  @action
+  loading(transition) {
+    const to = transition.to?.name;
+    return to === 'scopes.scope.roles.role.index';
```

Comment:
I think you need to check for both `from` and `to`, this seems to prevent a loading indicator going from the roles to the individual role route

Reply:
im probably misunderstanding but when going from `roles` to `role` i am seeing the loading screen.

https://github.com/user-attachments/assets/c5f720f9-cda5-4bae-9eee-fa2206f7836a

Reply:
Woops yeah I'm seeing it now, I think it was bc I forgot I moved the scopes loading to `roles` from `roles.index` for testing which changed the logic, looks fine now! Sorry false alarm!

But the other thing I'm noticing is no loading screens for any of the manage-scope screens, do you see that too?

Reply:
ah yup ill fix that